README.md

```diff
@@ -1,14 +1,82 @@
-# Anthropic Xircuits Component Library
+<p align="center">
+  <a href="https://github.com/XpressAI/xircuits/tree/master/xai_components#xircuits-component-library-list">Component Libraries</a> •
+  <a href="https://github.com/XpressAI/xircuits/tree/master/project-templates#xircuits-project-templates-list">Project Templates</a>
+  <br>
+  <a href="https://xircuits.io/">Docs</a> •
+  <a href="https://xircuits.io/docs/Installation">Install</a> •
+  <a href="https://xircuits.io/docs/category/tutorials">Tutorials</a> •
+  <a href="https://xircuits.io/docs/category/developer-guide">Developer Guides</a> •
+  <a href="https://github.com/XpressAI/xircuits/blob/master/CONTRIBUTING.md">Contribute</a> •
+  <a href="https://www.xpress.ai/blog/">Blog</a> •
+  <a href="https://discord.com/invite/vgEg2ZtxCw">Discord</a>
+</p>
 
-This library contains Xircuits components for interacting with the Anthropic API, allowing you to easily integrate Anthropic's functionalities, such as generating text, into your Xircuits projects. 
 
 
-## Installation
 
-```bash
-pip install -r requirements.txt
-```
 
-To use this component library, simply copy the directory / clone or submodule the repository to your working Xircuits project directory.
+<p align="center"><i>Xircuits Component Library to interface with 
+Anthropic AI! Build AI-powered solutions.</i></p>
+
+---
+## Welcome to the Anthropic XAI Components Library
```

Comment:
Instead of Welcome I think this should be "Xircuits Component Library for Antropic"

Reply:
okay